prometheus/base/alerting/prometheus.argoCD_alerts.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-argoCD-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: argoCD_alerts
+    rules:
+    - alert: DegradedArgoCDApp
+      expr: max_over_time(argocd_app_info{health_status="Degraded", dest_namespace!~".+-tenant"}[5m]) == 1
+      for: 15m
```

Comment:
I'm trying to wrap my head around what will be the outcome of the combination of max over the last 5 minutes together with for the last 15 minutes.

I think it means that it will only be alerting if the status was degraded at some point at every 5 minutes window within the last 15 minutes. It's a bit convoluted for me. Is this what we wanted? Can we make it any simpler?

Reply:
so you suggest we change that to also 5 minutes?

Reply:
It is now 5m